cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -389,7 +389,17 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> Tuple[str
         return ("|",) * self.num_qubits()
 
 
-def barrier(*qubits: cirq.Qid) -> cirq.Operation:  # pylint: disable=missing-function-docstring
+def barrier(*qubits: cirq.Qid) -> cirq.Operation:
+    """Cirq equivalent of
+    https://qiskit.org/documentation/stubs/qiskit.circuit.library.Barrier.html.
```

Comment:
Nit: this could fit on one line if you got rid of https://
```suggestion
    """Cirq equivalent of qiskit.org/documentation/stubs/qiskit.circuit.library.Barrier.html.
```

Reply:
Per dev guide, the format of docstring should be one sentence, blank new line, then additional description. (So there should be a blank new line on L394)

Reply:
(if you want to keep the https we could alternatively maybe get rid of the "Cirq", which should be clear from this being cirq-superstaq)
```suggestion
    """Equivalent to https://qiskit.org/documentation/stubs/qiskit.circuit.library.Barrier.html.
```